user/cron-jobs.md

```diff
@@ -27,6 +27,10 @@ Confirm that the cron job is displayed in your settings tab:
 
 ![cron job created](/images/cron-created.png "cron job created")
 
+## Skipping Cron Jobs
+
+Currently, a new cron job looks to the most recent commit for whether [`[ci skip]` or `[skip ci]`](/user/customizing-the-build/#Skipping-a-build) is included in the commit message. If the commit message contains at least one of these, the cron job will not be run. 
```

Comment:
Agreed! I'm working on getting it in the opening section of the description of cron jobs, but without restructuring the paragraph too much. Agreed that the `[skip ci]` note should come later.